src/clr-angular/forms/select/_select-angular.clarity.scss

```diff
@@ -52,6 +53,19 @@
 
     .clr-option {
         @include generate-dropdown-item();
+
+        white-space: nowrap;
```

Comment:
I think this is a problem. Custom options are going to be multi-line, and we don't want them to have to overwrite this every single time. Why do you need it? Can't we simply let the option text wrap?

Reply:
I believe this is what we do for dropdowns. Custom options can still be online though. They would be in a separate container of their own. That container by default can wrap unless specified not to by the user, no?

Reply:
`white-space` is inherited, so that's what I meant by having to override it. I think that's bad for DX.

Reply:
removed for now but we can discuss later if i decide to add it back.